docs/examples/usecases/train_neural_network.ipynb

```diff
@@ -38,15 +38,28 @@
     "First, we'll load the dataset from `scikit-learn`. The Iris Dataset contains 3 classes for each of the iris species (_iris setosa_, _iris virginica_, and _iris versicolor_). It has 50 samples per class with 150 samples in total, making it a very balanced dataset. Each sample is characterized by four features (or dimensions): sepal length, sepal width, petal length, petal width."
```

Comment:
<p>Thanks for putting things in the proper headers! it looks more readable now :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ljvmiranda921/pyswarms/pull/379/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>